### PR TITLE
feat: create template pull request

### PR DESCRIPTION
**Pull request by TranQuangPhi**
## Summary
- add template pull request

## Features
- None

## Bugs
- None

## Check List
- [x] Your code builds clean without any errors or warnings.
- [x] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
- None

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Summary
+
```
Comment (loc-nguyenthien) [inline .github/pull_request_template.md:2]: Add some description in these field for user to edit (noted to remove if unnecessary)

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]

**Review by loc-nguyenthien — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+## Features
+
+
+## Bugs
+
+
+## Check List
+- [ ] Your code builds clean without any errors or warnings.
+- [ ] Check coding convention.
```
[the remark this answers is not in the document]
Comment (TranQuangPhi) [reply .github/pull_request_template.md:12]: added

**Review by ThinhKimVo — APPROVED** [review]

**Review by honghoangsts — APPROVED** [review]
